lesson_04_homework.rb

```diff
@@ -4,34 +4,59 @@ def fizzbuzz(num)
   # Example comment line
 
   case
+    #statement 
     when num % 15 == 0 then "FizzBuzz"
+      #every 15th number types "fizzBuzz"
     when num % 3  == 0 then "Fizz"
+      #every 3rd number types "Fizz"
     when num % 5  == 0 then "Buzz"
+      #every 5th number types "Buzz"
     else num
+      #otherwise just prints the number
   end
 end
 
 def fizz_buzz_to(limit)
+  #putting fizz_buzz as a method and giving it a limit variable 
   1.upto(limit).each do |num|
+    #goes through the upto erray and executes each code element in it, adds +1
     puts fizzbuzz(num)
+    #prints 
   end
+    #end of loop
 end
+  #end method
 
 # Fix Broken Code
 
-def area_of_triangle(b, h)
-  puts b x height / 2
+def area_of_triangle(base, height)
+  base * height / 2
 end
 
 base = 7
 height = 6
 
-puts area_of_triangle(height, height)
+puts area_of_triangle(base, height)
 
 # Coding
 
 class Waitlist
-  # add your code here
+  def initialize(parties)
+    @parties = parties 
+  end
+ 
+  def list
+    @parties  
+  end  
+
+  def add_party(i)
+    parties << i
+  end
+  
+  def seat
```

Comment:
@loriksm while this returns the first element in the array, only issue is that you are not removing the element from the parties array. Check the answer (will be posted later).